aviatrix/resource_aviatrix_transit_gateway.go

```diff
@@ -753,6 +753,31 @@ func resourceAviatrixTransitGateway() *schema.Resource {
 					},
 				},
 			},
+			"interface_mapping": {
+				Type:        schema.TypeList,
+				Optional:    true,
+				Description: "List of interface names mapped to interface types and indices.",
+				Elem: &schema.Resource{
+					Schema: map[string]*schema.Schema{
+						"name": {
+							Type:        schema.TypeString,
+							Required:    true,
+							Description: "Interface name (e.g., 'eth0', 'eth1').",
+						},
+						"type": {
+							Type:         schema.TypeString,
+							Required:     true,
+							Description:  "Interface type (e.g., 'wan', 'mgmt').",
```

Comment:
The examples should exactly match what you validate for

Reply:
Added the change in validation function